tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,78 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def adding_label(instance):
+    """
+    Method add multiple labels in the node
+    Args:
+        instance: cluster instance
+        node: cluster node
+    Returns:
+        boolean
+    """
+
+    node = instance.cluster.get_nodes()[0].hostname
+    out, _ = instance.installer.exec_command(
+        cmd="cephadm shell ceph orch host add "
+        + node
+        + " "
+        + node
+        + " node-exporter crash osd installer grafana prometheus alertmanager",
+        sudo=True,
+    )
+    if "Added host" in out:
+        log.info("Successfully added multiple roles in same admin node ")
+        return 0
+    log.info("Adding multiple roles in same admin node fail")
+    return 1
+
+
+def run(ceph_cluster, **kw):
+    """Verify removal of _admin node acting as a bootstrap node
+    Args:
```

Comment:
This description is not necessary, it's mandatory function to have for test, based on general acceptance.

Reply:
Thanks @vamahaja
Removed